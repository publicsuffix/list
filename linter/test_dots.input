// test:
// - leading dot
// - trailing dot
// - consecutive dots

// ===BEGIN ICANN DOMAINS===

// example.com: https://www.iana.org/domains/reserved
.a.example.com
b.example.com.
c..example.com

// ===END ICANN DOMAINS===
