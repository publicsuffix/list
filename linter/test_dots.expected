9: error: Leading/trailing or multiple dot: '.a.example.com'
10: error: Leading/trailing or multiple dot: 'b.example.com.'
11: error: Leading/trailing or multiple dot: 'c..example.com'
13: warning: No PRIVATE section found
